handlers/network_data_aggregator.py

```diff
@@ -0,0 +1,12 @@
+import redis
+import json
+
+redis_client = redis.Redis(host='localhost', port=6379, db=0)
+
+def handle(node_status: dict, _context):
```

Comment:
Missing `_context` type, return type, and docstring. We shouldn't use `dict` but `Dict[key_type, value_type]`. I also think we should avoid using dict at all costs and always replace by a specific class or namedtuple.